tests/validators/valid_urls.txt

```diff
@@ -48,7 +48,6 @@ http://foo.bar/?q=Test%20URL-encoded%20stuff
 http://مثال.إختبار
 http://例子.测试
 http://उदाहरण.परीक्षा
-http://-.~_!$&'()*+,;=:%40:80%2f::::::@example.com
```

Comment:
Could we leave this, but just remove the offending `::::::`?

Reply:
One of the other colons (in `=:%40` or `%40:80`) would also need to be removed for this test case to pass.